code/__DEFINES/mobs.dm

```diff
@@ -528,7 +530,7 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define HUNTER_SILENCE_WINDUP 0.5 SECONDS //Windup of the Hunter's Silence
 #define HUNTER_SILENCE_MULTIPLIER 1.5 //Multiplier of stacks vs Hunter's Mark targets
 #define HUNTER_SILENCE_WHIFF_COOLDOWN 3 SECONDS //If we fail to target anyone with Silence, partial cooldown to prevent spam.
-
+#define HUNTER_VENT_CRAWL_TIME 1 SECONDS //Hunters can enter vents fast
```

Comment:
Make it 2 seconds for hunters

Reply:
Eh, I tried it, seems pretty slow, and 1 is still enough of a delay that they can be interrupted/killed

Reply:
This is done